docusaurus.config.js

```diff
@@ -222,6 +224,51 @@ module.exports = {
            */
           // excludeNextVersionDocs: false,
           includeCurrentVersion: true, // excludeNextVersionDocs is now deprecated
+          remarkPlugins: [
+            [
+              () =>
+                function addTSNoCheck(tree) {
+                  // Disable TS type checking for any TypeScript code blocks.
+                  // This is because imports are messy with snipsync: we don't
+                  // have a way to pull in a separate config for every example
+                  // snipsync pulls from.
+                  function visitor(node) {
+                    if (!/^ts$/.test(node.lang)) {
+                      return;
+                    }
+                    node.value = "// @ts-nocheck\n" + node.value;
```

Comment:
What a hack, I wish there was a better way around it, can't think of one though.

Reply:
So you want to transpile, but not get any errors? 
I would also accept a PR for something like that over in the remark plugin.

Reply:
@phryneas there's a couple issues why I'm adding `// @ts-nocheck`: 1) setting up all the dependencies has proven to be tricky, especially since `@activities` can refer to different files depending on where it is being imported from, 2) some samples would be too verbose if we include all the boilerplate, so we'd like the option to include smaller snippets that may omit some code that the compiler would think is necessary.

Reply:
Oh, okay :)

Generally, you can also add some boilerplate-like code as files and include that from all your examples, but if it's tested already it'll probably be fine.

Reply:
I'm fine with this because we make sure to test the codebase.